deploy/charts/cert-manager/templates/_helpers.tpl

```diff
@@ -157,3 +157,12 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 helm.sh/chart: {{ include "chartName" . }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Namespace for all resources to be installed into
+If not defined in values file then the helm release namespace is used
+By default this is not set so the helm release namespace will be used
+*/}}
+{{- define "cert-manager.namespace" -}}
+    {{ .Values.namespace | default .Release.Namespace }}
+{{- end -}}
```

Comment:
Please add a newline here.

Reply:
Do you mean a new line at the bottom of the file? So the last line is a blank line?

Reply:
@wallrj just checking you saw my comment here? Wanted to check what you meant?